### PR TITLE
SOLR-17074 - Not correctly escaped quote in bin/solr script

### PR DESCRIPTION
**Pull request by freedev**
https://issues.apache.org/jira/browse/SOLR-17074

# Description

fixed the script `solr/bin/solr` 

# Solution

Added the missing quotes 

# Tests

Tried to run the script locally, now the output is correct. 

# Checklist

Please review the following and check all that apply:

- [x] I have reviewed the guidelines for [How to Contribute](https://github.com/apache/solr/blob/main/CONTRIBUTING.md) and my code conforms to the standards described there to the best of my ability.
- [ ] I have created a Jira issue and added the issue ID to my pull request title.
- [ ] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [x] I have developed this patch against the `main` branch.
- [x] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

File: solr/bin/solr [hunk 1]
```diff
@@ -422,7 +422,7 @@ function print_usage() {
     echo ""
     echo "  -noprompt             Don't prompt for input; accept all defaults when running examples that accept user input"
     echo ""
-    echo "  -force                If attempting to start Solr as the root user, the script will exit with a warning that running Solr as "root" can cause problems."
+    echo "  -force                If attempting to start Solr as the root user, the script will exit with a warning that running Solr as \"root\" can cause problems."
     echo "                          It is possible to override this warning with the `-force` parameter."
```
Comment (cpoerschke) [inline solr/bin/solr:426]:
```suggestion
    echo "                          It is possible to override this warning with the '-force' parameter."
```

**Review by cpoerschke — COMMENTED** [review]
Thanks @freedev for contributing this pull request!

Would you also like to add an entry in the 9.5.0 Bug Fixes section of `solr/CHANGES.txt` for this change?

File: solr/bin/solr (hunk 1 above)
Comment (epugh) [reply solr/bin/solr:426]: i like this change..

Comment (freedev) [reply solr/bin/solr]: I've added in my PR

File: solr/CHANGES.txt [hunk 2]
```diff
@@ -174,6 +174,8 @@ Bug Fixes
 
 * SOLR-16203: Properly initialize schema plugins loaded by SPI name (janhoy, hossman, Uwe Schindler)
 
+* SOLR-17074: Fixed not correctly escaped quote in bin/solr script (Dominique Béjean, Vincenzo D'Amore)
+
```
Comment (cpoerschke) [inline solr/CHANGES.txt:178]: Thanks @freedev for iterating on this pull request! I've moved the entry to the 9.5 section here and added attribution for both yourself and Dominique as reporter of the issue.

Please let us know if you'd prefer a different attribution e.g. see line 175 above, some folks use their handle and others their name.

**Review by cpoerschke — APPROVED** [review]

**Pull request merged**